publication-events/src/test/java/no/unit/nva/publication/events/PublicationEventsTestConfig.java

```diff
@@ -0,0 +1,13 @@
+package no.unit.nva.publication.events;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import nva.commons.core.JsonUtils;
+
+public final class PublicationEventsTestConfig {
```

Comment:
if the objectMapper is used only in tests, I create BlablaTestConfig for all the tests.